Add option to export data from maintainance tab

destrex271 opened the pull request:
As mentioned in : #2432 , this PR introduces an export option in the maintenance tab. Currently added export option to get all subscribers. Still in progress

destrex271 committed:
added export button to subscribers

destrex271 committed:
Added export all campaign views; needs refactoring

destrex271 committed:
fixed campaign views export option

destrex271 committed:
added link clicks export option

destrex271 committed:
fixed lint errors

destrex271 committed:
fixed lint errors

destrex271 committed:
added separation between export and delete

destrex271 committed:
added campaign select box and removed logs

destrex271 committed:
updated query to return subscribed or unsubscribed based on value

destrex271 commented:
Hi,
This pull request is ready for review.

Export Analytics:

![image](https://github.com/user-attachments/assets/0ed0fc97-86c2-4d1a-94d7-6916b00285a6)


Export All Subscribers:

![image](https://github.com/user-attachments/assets/69e50bdb-e3ef-484c-beb7-fa76e9683b94)

destrex271 commented:
Hi! Just following up on this PR.

knadh commented:
Hi @destrex271.

All subscribers (or selected subscribers) can already be exported from the subscribers page.

<img width="824" height="572" alt="image" src="https://github.com/user-attachments/assets/dffd2720-74a2-4db4-8fd4-13f2468f661c" />

Can you amend the PR to remove this from the maintenance page? We can consider the export option for `Analytics`

destrex271 commented:
Oh ok, I missed that.

Will do this asap

destrex271 committed:
Merge branch 'master' into feature/csv_export

destrex271 committed:
removed export option from all subscribers view

destrex271 commented:
Hi @knadh , updated this PR. 

I have removed the exportSubscribers function and the anchor tag enclosing the Export option from Subscribers view. Apologies for the delay!

destrex271 committed:
fix: restored subscriber view page

destrex271 committed:
removed subscribers export option from maintenance view

destrex271 commented:
Restored the export function on subscribers list page. 
Removed from maintenance tab.
